templates/account/update-customer-to-vendor.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Dokan Update Customer to Vendor Template.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @package dokan
+ */
+$user_id   = get_current_user_id();
```

Comment:
can you please move all the validation to the corresponding method and pass the variables from there?